README.md

```diff
@@ -619,6 +621,131 @@ When enabled, the overall patient-level results will contain the raw results of
 ];
 ```
 
+## Composite Measures
```

Comment:
Since we now have support for composite measures, we should probably note somewhere in this section what fqm-execution functions still do *not* support composite measures.